core/src/Cuda/Kokkos_Cuda_GraphNodeKernel.hpp

```diff
@@ -34,6 +34,35 @@
 namespace Kokkos {
 namespace Impl {
 
+template <typename Functor>
+struct GraphNodeCaptureImpl<Kokkos::Cuda, Functor> {
+  Functor m_functor;
+  cudaGraphNode_t m_node = nullptr;
+
+  void capture(const Kokkos::Cuda& exec, cudaGraph_t& graph) {
```

Comment:
```suggestion
  void capture(const Kokkos::Cuda& exec, cudaGraph_t graph) {
```

It appears the `graph` is passed by value in other cases such as

https://github.com/kokkos/kokkos/blob/8cd597c7a7119f5984415db2b6f69d0ca38825b0/core/src/Cuda/Kokkos_Cuda_Instance.hpp#L281-L285

Reply:
I makes sense since it's an opaque pointer. Done :heavy_check_mark: